include/range/v3/view/iota.hpp

```diff
@@ -524,16 +543,17 @@ namespace ranges
                 detail::take_exactly_view_<iota_view<Val>, true>>
             ints_fn::operator()(Val from, Val to) const
             {
-                return {iota_view<Val>{from}, detail::iota_minus_(to, from)};
+                return {iota_view<Val>{from}, detail::ints_distance_(from, to)};
             }
 
             struct closed_ints_fn
             {
                 template<typename Val,
                     CONCEPT_REQUIRES_(Integral<Val>())>
+                RANGES_DEPRECATED("view::closed_ints is deprecated; use view::closed_indices instead!")
                 detail::take_exactly_view_<iota_view<Val>, true> operator()(Val from, Val to) const
                 {
-                    return {iota_view<Val>{from}, detail::iota_minus_(to, from) + 1};
+                    return {iota_view<Val>{from}, detail::ints_distance_(from, to) + 1};
```

Comment:
If we're always doing `n+1` in `closed_ints` and `inclusive_indices`, then the need for the view goes away. Users can do `+1` themselves. What they *can't* do it get a closed range that covers the entire range of integers without fear of overflow.

If we decide that's not a valid thing to want, I suggest we deprecate `closed_ints` and `inclusive_indices`. (And I disapprove of the inconsistent naming, FWIW.) If it *is* a valid thing to want, then this PR is wrong.